mathesar/api/exceptions/exceptions.py

```diff
@@ -0,0 +1,46 @@
+from collections import namedtuple
+
+from django.utils.encoding import force_str
+from rest_framework import status
+from rest_framework.exceptions import APIException as DRFApiException
```

Comment:
this should be `DRFAPIException` or something like `OriginalAPIException` if the concatenated acronyms are hard to read, it's confusing to mix `API` and `Api`